### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sagarkar10**
Added the absolute path format to `find` command

### CODE REVIEW

**Comment by sagarkar10** [comment]
@trimstray please review!

File: README.md [hunk 1]
```diff
@@ -1276,6 +1276,12 @@ find . -type f -exec stat --format '%Y :%y %n' "{}" \; | sort -nr | cut -d: -f2-
 find . -not -path '*/\.git*' -type f -print0 | xargs -0 sed -i 's/foo/bar/g'
 ```
 
+###### find with the absolute path shown and may be grep for something
```
Comment (fbartels) [inline README.md]: better title would be. "List all files below the current folder"

**Comment by trimstray** [comment]
Ok, but it doesn't do anything special. Maybe something more for hackers? :) You know - something rarely seen.

On the other hand, why this `one-liner` is good for you? Look at [this](https://stackoverflow.com/questions/246215/how-can-i-generate-a-list-of-files-with-their-absolute-path-in-linux).

**Comment by sagarkar10** [comment]
Ok

**Pull request closed without merging**